tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -euo pipefail
```

Comment:
What does this do? (i am a bash n00b haha) Maybe add a comment?

Reply:
This sets bash's error handling. Specifically:

`-e`: Any line failing causes the whole script to fail
`-u`: Using an undeclared variable raises an error
`-o pipefail`: Respect the exit code of any command that fails within a piped expression

Screenshot of `pipefail` in action. You'll notice that initially, `false | echo ok` uses the exit code of `echo ok`, which is 0. After calling `set -o pipefail` it now respects the exit code of `false`, which is 1.
<img width="495" alt="Screen Shot 2020-10-14 at 9 46 33 PM" src="https://user-images.githubusercontent.com/8296030/96078303-c5a04480-0e66-11eb-8034-bb0de7eed48d.png">

Reply:
It all makes sense now :)